sci-libs/libticables2/libticables2-1.3.5.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Library to handle different link cables for TI calculators"
+HOMEPAGE="http://lpg.ticalc.org/prj_tilp/"
+SRC_URI="mirror://sourceforge/tilp/tilp2-linux/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE="debug doc nls static-libs usb"
+
+RDEPEND="
+	dev-libs/glib:2
+	usb? ( virtual/libusb:1 )
+	nls? ( virtual/libintl )"
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	nls? ( sys-devel/gettext )"
+
+DOCS=( AUTHORS LOGO NEWS README ChangeLog docs/api.txt )
+
+src_prepare() {
+	default
+	eautoreconf
```

Comment:
If I remember correctly, in all ti* packages, no configure script is provided.
In any case, the build failed without eautoreconf